src/components/OptionsWrapper.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div :class="className" class="w-100 h-100">
+    <app-toolbar-centered app :title="title" show-back flat sticky no-max-width />
+
+    <v-container fluid class="px-0 py-0 container--with-app-toolbar">
```

Comment:
Let's get rid of this class `container--with-app-toolbar`. Better to import and use `--toolbar-height`.

https://github.com/Adamant-im/adamant-im/blob/dev/src/assets/styles/generic/_layout.scss#L6-L8